qcengine/programs/entos.py

```diff
@@ -90,6 +121,27 @@ class EntosHarness(ProgramHarness):
         "xtb": _xtb_keywords_extra,
     }
 
+    # This map order converts entos ordering to CCA ordering
+    # Entos spherical basis ordering for each angular momentum
+    # S -->  0
+    # P --> +1,  0, -1
+    # D --> +2, +1,  0, -1, -2
+    # F --> +3, +2, +1,  0, -1, -2, -3
+    # G --> +4, +3, +2, +1,  0, -1, -2, -3, -4
+    # H --> +5, +4, +3, +2, +1,  0, -1, -2, -3, -4, -5
+    # I --> +6, +5, +4, +3, +2, +1,  0, -1, -2, -3, -4, -5, -6
+    _entos_to_cca_ao_order: Dict[str, Dict[int, List[int]]] = {
```

Comment:
It this is just a reversal, maybe add an automatically generating one for arbitrary angular momentum? Granted unlikely to go past 6, but best not to artificially cap.

Reply:
Yes this is just a reversal. And sounds good!